ansible/files/bootstrap/consul/post/test/vault-integration.yml

```
@@ -0,0 +1,78 @@
+# test admin.team.policy
+#TODO: Use consul client for this??
+- name: "{{ service }} - post/test/vault-integration - admin token - write key-value to root '/' is forbidden"
+  uri:
+    # curl -v -X PUT --header "X-Consul-Token: <token>" -d 'my-admin-value' http://127.0.0.1:8500/v1/kv/my-admin-key
+    url:  http://127.0.0.1:8500/v1/kv/my-admin-key
+    method: PUT
+    body: "my-admin-value"
+    status_code: 403
+    return_content: yes
+    headers:
+      X-Consul-Token: "{{ consul.acl.admin_team.token }}"
+
+- name: "{{ service }} - post/test/vault-integration - admin token - write key-value to /team/admin is allowed"
+  uri:
+    # curl -v -X PUT --header "X-Consul-Token: <token>" -d 'my-admin-value' http://127.0.0.1:8500/v1/kv/team/admin/my-admin-key
+    url:  http://127.0.0.1:8500/v1/kv/team/admin/my-admin-key
+    method: PUT
+    body: "my-admin-value"
+    status_code: 200
+    return_content: yes
+    headers:
+      X-Consul-Token: "{{ admin_team_token.stdout }}"
```

```suggestion
      X-Consul-Token: "{{ consul.acl.admin_team.token }}"
```
This change needs to be made in the rest of the ansible tasks that make use of **admin_team_token**.

---

I would like to offer to delete `admin-team` and `dev-team` policies, due that we will change it with another strategy Admin/Producer/Consumer https://github.com/fredrikhgrelland/vagrant-hashistack-template/issues/33